zerver/tests/test_subs.py

```diff
@@ -575,6 +579,7 @@ def test_create_stream_by_admins_only_setting(self):
 
         # Change setting back to default
         do_set_realm_create_stream_by_admins_only(user_profile.realm, False)
+        do_set_realm_waiting_period_threshold(user_profile.realm, initial_threshold)
```

Comment:
@timabbott i have added the test in the commit. The updated commit is not visible here. Please review this [PR](https://github.com/zulip/zulip/pull/2479).